src/main/java/com/ververica/flink/table/gateway/context/ExecutionContext.java

```diff
@@ -210,6 +211,12 @@ public ClusterID getClusterId() {
 		return clusterClientFactory.createClusterDescriptor(flinkConfig);
 	}
 
+	public ClusterID getClusterId(String appId) {
```

Comment:
It might be better to add a `setClusterId` method so that we can unify the logic in `AbstractJobOperation`.

Reply:
Got it. I will refactor these codes.